src/metadatas.rs

```diff
@@ -15,9 +15,14 @@ pub struct Metadata {
     pub issues_count: std::collections::BTreeMap<IssueType, usize>,
     pub has_fares: bool,
     pub has_shapes: bool,
+    // some stops have a pickup_type or drop_off_type equal to "ArrangeByPhone"
+    pub some_stops_need_phone_agency: bool,
```

Comment:
I wonder if naming after the original property would help understanding the codebase later?

E.g. `some_stops_need_arrange_by_phone`, `some_stops_need_coordinate_with_driver`

Reply:
hum not a strong opinion on this, but I find the names as they are quite understandable.